e2e/require-missing-ext/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "jest": {
+    "moduleFileExtensions": [
+      "js"
+    ],
+    "testEnvironment": "node"
+  },
+  "dependencies": {
+    "discord.js": "11.5.1"
```

Comment:
this is a 180kB+ lib, can we add something smaller?

Reply:
we just need something that does a bajillion `require`s synchronously (otherwise the `stackTraceLimit` change doesn't change anything). Suggestions?

It's also just in an e2e test, so not a big issue if it's hefty, imo

Reply:
That's fine then 👍